Cargo.lock

```diff
@@ -4,2125 +4,2123 @@
 name = "adblock"
 version = "0.3.1"
 dependencies = [
- "addr 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Not sure what you did to the `Cargo.lock` file - would you mind reverting the changes here?

Reply:
I'm not sure either! I've reverted the changes.